app/controllers/api/server_time_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  class ServerTimeController < ApplicationController
+    
+    def server_time
+      time = Time.current()
+      render json: { data: {time: time} }
```

Comment:
Does this return a [http status](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status#informational_responses)? These are very helpful in enforcing a contract between the clients and server. 
The third acceptance criteria for the task was `Write a test that ensures the endpoint when called has a 200 status`, so once you have updated the controller to answer a https status cpde, also try to add a `request` test that confirms that when this endpoint is called it answers a `200` status code